server/server.go

```diff
@@ -378,13 +391,26 @@ func (s *Server) v0DataPost(w http.ResponseWriter, r *http.Request) {
 
 func (s *Server) v0QueryPath(w http.ResponseWriter, r *http.Request, path ast.Ref) {
 	ctx := r.Context()
+	webhook := path.Equal(systemMainPath)
+
+	var settings settings
+	if webhook {
+		settings = s.evalDiagnosticPolicy(r)
+	}
 
 	input, err := readInputV0(r.Body)
 	if err != nil {
 		writer.ErrorString(w, http.StatusBadRequest, types.CodeInvalidParameter, errors.Wrapf(err, "unexpected parse error for input"))
 		return
 	}
 
+	var goInput interface{}
+	if input != nil {
+		if goInput, err = ast.ValueToInterface(input, stringResolver{}); err != nil {
+			panic("not reached")
```

Comment:
I don't see a reason to panic here. The server can return HTTP 500 if this occurs.